addons/mail/static/src/utils/common/format.js

```diff
@@ -323,3 +317,43 @@ export function parseEmail(text) {
 }
 
 export const EMOJI_REGEX = /\p{Emoji_Presentation}|\p{Emoji}\uFE0F|\u200d/gu;
+
+/**
+ * Wrap emojis present in the given text with a title and return a safe HTML
+ * string.
+ *
+ * @param {string|ReturnType<markup>} text
+ * @returns {ReturnType<markup>}
+ */
+export function wrapEmojisWithTitles(text) {
+    if (!loader.loaded) {
+        throw new Error("'wrapEmojisWithTitles' requires emoji data to be loaded.");
+    }
```

Comment:
Do we really need to crash? It could just return original content and rely on reactive to trigger the compute again?